src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/PgQuestionAttempts.java

```diff
@@ -438,9 +445,10 @@ public Map<Date, Long> getQuestionAttemptCountForUserByDateRange(final Date from
     private LightweightQuestionValidationResponse resultsToLightweightValidationResponse(final ResultSet results) throws SQLException {
         LightweightQuestionValidationResponse partialQuestionAttempt = new QuestionValidationResponse();
 
-        partialQuestionAttempt.setCorrect(results.getBoolean("correct"));
+        partialQuestionAttempt.setCorrect(results.getInt("marks") > 0);
```

Comment:
This isn't necessarily safe, if `marks` could be `NULL` in the database. But it might be that it coerces `NULL` to 0, in which case it is safe, I am not 100% sure.

However, I would still load `correct` here, rather than using `marks`. That seems safer and we can remove `correct` as an independent thing later on. We want to start writing `marks` for all question attempts before we start any migrations, so loading this from `marks` will break old attempts too soon.